network/netplan/netplan.go

```diff
@@ -195,11 +215,16 @@ func (np *Netplan) Write(inPath string) (filePath string, err error) {
 	} else {
 		filePath = inPath
 	}
+	tmpFilePath := fmt.Sprintf("%s.tmp.%d", filePath, time.Now().UnixNano())
```

Comment:
there are things like https://golang.org/pkg/io/ioutil/#TempFile
so that you don't have to think about this as much.
Though we'd have to be careful about what the permissions are on that final file.